_drafts/coop-values.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Coop Community Values"
+author: Lauren Wolfe
+categories: # delete the categories that do not apply and keep at least one
+  - Community
+tags:
+  - community values
+---
+
+Having gone through a few [transitions]() in recent months along with global and local conversations surrounding the COVID-19 pandemic and rampant anti-blackness in the United States have caused those of us at the Coop to take a step back and really consider why we do what we do. From these discussions we have seen a need for and are beginning to develop the Coop's Community Values. 
+
+For full transparency and to ensure we capture the actual values of our community members we will be holding a community conversation in the next week where community members can bring their thoughts and suggestions to the table. Following this we will formalize our values statements and release a document for public comment.
+
+## Why Develop Community Values?
+
+Community values are a direct response to the theme of individualism that is prevelant in American and academic culture. Individualism is a narrative that celebrates the successes of the individual over the common good shifts the blame for falling behind to the individual rather than examining root causes and systematic barriers to success. Policy rooted in individualism favor solutions that leave individual people to figure out solutions on their own rather than addressing challenges as a collective. 
```

Comment:
I think you need a clear definition of community values before going into individualism: community values frame what we do and how we do it. (or something similarly vernacular)